README.md

```diff
@@ -16,9 +16,9 @@ This sample code is provided for illustrative purposes only. Any use of this rep
 0) Download the files
 1) Navigate to directory location
 3) Run composer ```php composer.phar install```
-4) Rename file ```.env.example``` into ```.env``` and fill the Cardinity API credentials
-5) Start internal PHP server ```php -S localhost:8000 -t public```
-6) Launch browser with url ```http://localhost:8000```
+4) Rename file ```.env.example``` into ```.env``` and fill the Cardinity API credentials and your base URL, localhost is restricted in our gateway so use IP address or fake domain
```

Comment:
I don't think this server start configuration relates to our SDK restricting localhost addresses. The callback URL is set in `BASE_URL` field in `.env`, so the non-localhost URL should be set there